tests/README.md

```diff
@@ -19,5 +19,11 @@ There is a helper script to run this:
 
 ```
 # from the git repo root folder
-cluster/run_tests.sh
+hack/build-go.sh functest
```

Comment:
I think `make functest` is good enough. Is there any reason why I should prefer this over the make target?